src/components/wayfinder/wayfinder.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import data from "../../data/wayfinderData.js";
```

Comment:
I think we will find just 'data' to be a bit vague for naming. For example, when we track user's answers to send GP,  this might be data as well. No need to change now, just bear in mind.